app-libs/stf/src/getter.rs

```diff
@@ -209,22 +209,18 @@ impl ExecuteGetter for Getter {
 				TrustedGetter::pay_as_bid_proof(_who, timestamp, actor_id) => {
 					let now = Instant::now();
 
-					// log the error and return none.
-					if let Ok((orders, index)) = get_orders_index(timestamp, &actor_id) {
-						let proof: MerkleProofWithCodec<_, _> =
```

Comment:
I must admit, the let else this not work exactly the way I remembered here. The correct way to do this, would be a match statement, as I did here.